vm/src/import.rs

```diff
@@ -105,23 +105,37 @@ fn remove_importlib_frames_inner(
     vm: &VirtualMachine,
     tb: Option<PyTracebackRef>,
     always_trim: bool,
-) -> Option<PyTracebackRef> {
-    let traceback = tb.as_ref()?;
+) -> (Option<PyTracebackRef>, bool) {
+    if tb.is_none() {
+        return (None, false);
+    }
+    let traceback = tb.unwrap();
```

Comment:
I'd say you could do something like `let traceback = if let Some(tb) = { tb } else { return (None, false) };`